include/rocksdb/flush_manager.h

```diff
@@ -0,0 +1,33 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#pragma once
+
+#include <functional>
+#include <vector>
+
+namespace rocksdb {
+
+class FlushManager {
+ public:
+  virtual ~FlushManager() {}
```

Comment:
virtual not necessary.

Reply:
If it's not virtual, is it possible to lead to memory leaks in derived class objects? (Derived classes may potentially contain data members, though they do not now. Suppose the application deletes an pointer to `FlushManager`).